docs/pages/versions/unversioned/guides/using-styled-components.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Using Styled Components with Expo
+---
+
+Styled Components is a CSS-in-JS solution that enables you to create React components with a given style very easily. Using `styled-components` with Expo, you can create universal styles that'll work the same across web, mobile, and desktop!
+
+## Getting Started
+
+Install the package:
+
+```sh
+yarn add styled-components
+```
+
+Use `styled-components/native` instead of `styled-components`:
+
+```tsx
+import React from 'react';
+import styled from 'styled-components/native';
+
+const Container = styled.View`
+  flex: 1;
+  background-color: #fff;
+  align-items: center;
+  justify-content: center;
+`
+const Title = styled.Text`
+  color: #000;
+  text-align: center;
+  font-size: 1rem;
```

Comment:
maybe don't use `rem` if we want users to expect this code to work on native too?

Reply:
does rem not work on native?

Reply:
nope, they don't transform rem to pixels ![IMG_58432D989996-1](https://user-images.githubusercontent.com/12488826/73703301-4f76fb80-46a4-11ea-96bc-1188d4144380.jpeg)